### DISCUSSION TITLE
How to migrate VTM to mapsforge

### DISCUSSION THREAD

**Discussion by CodeK1988** [mapsforge/vtm#1103] [Q&A] [answered]
hi mate, how to migrate VTM to mapsforge?
I noticed that mapsforge seems to be updating all the time. and VTM seems to have given up on updating.
https://github.com/mapsforge/mapsforge

vtm vs mapsforge. what is the difference

**Comment by devemux86** [comment, marked as answer]
Both map libraries are continuously updated:

- Mapsforge: https://github.com/mapsforge/mapsforge

- VTM: https://github.com/mapsforge/vtm

VTM is an OpenGL map library with map rotation / tilt, continuous zoom, advanced themes and more.
It can read Mapsforge offline map files. See the [vtm-android-example](https://github.com/mapsforge/vtm/tree/master/vtm-android-example).
